airbyte-cdk/python/reference_docs/_source/conf.py

```diff
@@ -32,7 +32,10 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ["sphinx.ext.autodoc"]  # API docs
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.napoleon",  # Support for NumPy and Google style docstrings
```

Comment:
this is used so that in the docstrings of thedataclass `Attributes:` are displayed nicely on the page